src/pt/slimeread/src/eu/kanade/tachiyomi/extension/pt/slimeread/SlimeRead.kt

```diff
@@ -48,6 +50,29 @@ class SlimeRead : HttpSource() {
 
     private val json: Json by injectLazy()
 
+    private fun getApiUrlFromPage(): String {
+        val initClient = network.client
```

Comment:
why this? if you want a client without cloudflare interceptor, you woul have to manually remove it from the interceptors

Reply:
the client needs the apiUrl for ratelimit so it needs to use another